content/en/seachat/full-tutorials/01-mailerlite.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Mailerlite"
```

Comment:
Can you please update all mentions of Mailerlite to MailerLite(this is their official product name)